css-positioning/relative.css

```diff
@@ -0,0 +1,36 @@
+.pokemon {
+  width: 900px;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.pokemon > img {
+  width: 150px;
+
+  /* your code here */
```

Comment:
If you put position relative here, you don't have to put it in every selector below!